content/_index.md

```diff
@@ -136,7 +136,7 @@ title: Istio
     </div>
 
     <div id="buttons">
-        <a title="Install Istio on Kubernetes today." class="btn" href="/docs/setup/kubernetes/quick-start">GET STARTED</a>
+        <a title="Install Istio on Kubernetes today." class="btn" href="/docs/setup/kubernetes/install/">GET STARTED</a>
```

Comment:
This needs to link to quick start instructions, more like `/docs/setup/kubernetes/` instead of the `/docs/setup/kubernetes/install/`. The problem with `/docs/setup/kubernetes/`, however, is that since it's an _index page, you need to scroll down past all the sub document links to get to the important info that will help you figure out how to get started.

Reply:
I'm changing the link to `/docs/setup/kubernetes/` as suggested. We are looking into moving the sub document links to the bottom for certain pages but that is a separate effort.

Reply:
OK, but I think moving the getting started info to the top is critical. As it is, I didn't even notice it was there when I first clicked on the page.